packages/backend/src/controllers/slackController.ts

```diff
@@ -45,7 +44,9 @@ export class SlackController extends BaseController {
         if (!organizationUuid) throw new ForbiddenError();
         return {
             status: 'ok',
-            results: await slackClient.getChannels(organizationUuid),
+            results: await req.services.clients.slackClient.getChannels(
```

Comment:
imo the root problem is that the controller should call a service instead of a service. Once that is fixed we don't need to expose the `services.clients`.

Reply:
Added deprecation message

<img width="817" alt="Screenshot 2024-03-18 at 10 20 22" src="https://github.com/lightdash/lightdash/assets/9117144/1325594d-be29-4428-8bf1-1637e4739509">